src/lib/connectors/jsdom/evaluate-runner.ts

```diff
@@ -0,0 +1,41 @@
+import * as vm from 'vm';
+
+import * as jsdom from 'jsdom/lib/old-api';
+import * as jsdomutils from 'jsdom/lib/jsdom/living/generated/utils';
+
+const run = (data) => {
+    const { source, url } = data;
+    const result = {
+        error: null,
+        evaluate: 'result'
+    };
+
+    jsdom.env({
+        done: async (error, window) => {
+            if (error) {
+                result.error = error;
+
+                return process.send(result);
+            }
+
+            try {
+                const script: vm.Script = new vm.Script(source);
+                const evaluteResult = await script.runInContext(jsdomutils.implForWrapper(window.document)._global);
```

Comment:
After navigate you should wait for `this._options.waitFor` as it does in `jsdom.ts`. In that way, you will have a version of the site as much similar to the parent one as possible

Reply:
@sarvaje Done.